### ISSUE TITLE
fix websocket connections on rpc proxy

### ISSUE THREAD

**Issue by tomlinton** [coral-xyz/backpack#1723]
Ref https://github.com/coral-xyz/appstore/issues/44

**Comment by wentokay** [comment]
to be specific, this issue was fixed by #1988 i.e. wss://solana-rpc.backpack.xnfts.dev now works

but the swr proxy doesn't support websockets yet

**Comment by armaniferrante** [comment]
Got it. Is this coming up as required anywhere?

**Comment by wentokay** [comment]
Nope, I was using websockets to confirm a transaction with a CLI tool and noticed it error out. I don't think there's a need to increase the priority for backpack though, using `wss://solana-rpc.xnfts.dev` directly works ok if it's needed in these edge cases.
